.generated_files

```diff
@@ -0,0 +1,36 @@
+# Files that should be ignored by tools which do not want to consider generated
+# code.
+#
+# https://github.com/kubernetes/test-infra/blob/master/prow/plugins/size/size.go
+#
+# This file is a series of lines, each of the form:
+#     <type> <name>
+#
+# Type can be:
+#    path - an exact path to a single file
+#    file-name - an exact leaf filename, regardless of path
+#    path-prefix - a prefix match on the file path
+#    file-prefix - a prefix match of the leaf filename (no path)
+#    paths-from-repo - read a file from the repo and load file paths
+#
+
+file-prefix zz_generated.
+
+path-prefix vendor/
+path-prefix docs/source/api-reference/
+
+path-prefix pkg/api/
```

Comment:
This is too wide, API object structures and validation rules are written manually. 
There're couple of files autogenerated there (*.yaml)  but available matches won't be able to catch those.
Lets leave them accounted for size metric.

Reply:
It was initially so but I was asked in [this comment](https://github.com/scylladb/scylla-operator/pull/2526#discussion_r1979247435) to include `pkg/api/*/*` :grin:. Done - removed.

I think that the `zz_generated.` file-prefix should do a good enough job for the generated stuff under `pkg/api/*`.

Reply:
I got too excited before I got to the `.yaml` part and switched off my thinking :facepalm:. Thanks.